source/integrate-with-integration-environment/authenticate-your-user.html.md.erb

```diff
@@ -175,6 +175,8 @@ Use the guidance in the following table to replace placeholder values in your ex
 | `ui_locales`    | Optional             | GOV.UK One Login supports English and Welsh as language choices. <br><br> If your service is in Welsh, you may want to display GOV.UK One Login in Welsh for a consistent user experience. You can use `ui_locales` to do this. <br> <br> In the `ui_locales` parameter, you can choose either `en` (English) or `cy` (Welsh). <br><br> Using `ui_locales` is optional. If you do not include it, your service will continue using English by default. <br><br> GOV.UK One Login does not support any other languages. |
 | `vtr`           | Optional             | The `vtr` parameter represents ‘Vectors of Trust’ where you request authentication and, optionally, identity proving. For example, if you want the medium level of authentication and medium identity confidence, request `vtr=[“Cl.Cm.P2”]`. <br><br>You selected your Vector of Trust when you [chose the level of authentication][integrate.choose-level-of-auth] and [the level of identity confidence][integrate.choose-level-of-confidence] for your service.<br><br>You can read more about how to combine the vectors for authentication level and identity confidence in [Section 3 of RFC 8485](https://datatracker.ietf.org/doc/html/rfc8485#section-3.1). If you need identity proving, you must request `Cl.Cm` (the medium level of authentication).<br><br>If you do not specify the `vtr` parameter, your service will automatically log your users in at the medium level of authentication (`Cl.Cm`). This means you will not receive identity attributes in your response. |
 | `claims`        | Optional             | To get the identity attributes your service needs, you should specify these in the `claims` parameter using the `/userinfo` endpoint. The `/userinfo` endpoint returns a JSON object listing the requested claims.<br><br>You can read more about [choosing which user attributes your service can request][integrate.choose-user-attributes].<br><br>You can [read more about the structure of the claims request in OpenID Connect section 5.5](https://openid.net/specs/openid-connect-core-1_0.html#ClaimsParameter). |
+| `max_age`       | Optional             | `max_age` is only available to services not on the GOV.UK domain and those handling particularly sensitive data. Contact GOV.UK One Login support (https://www.sign-in.service.gov.uk/support) for more information. When the `max_age` parameter is included in your request, your user will be forced to reauthenticate if the time in seconds since authentication is greater than `max_age`. `max_age` must be set to zero or a positive integer. |
```

Comment:
```suggestion
| `max_age`       | Optional             | `max_age` is only available to services not on the GOV.UK domain and those handling particularly sensitive data. [Contact GOV.UK One Login support](https://www.sign-in.service.gov.uk/support) for more information. If you include the `max_age` parameter in your request, your user will be forced to re-authenticate if the time in seconds since authentication (shown as `auth_time` in the `/authorize` response) is greater than `max_age`. `max_age` must be set to zero or a positive integer. |
```

Reply:
Would recommend we give examples of what sensitive information might necessitate including max_age to stop too many services contacting us to find out. Which ones could work?

Reply:
have included an assumption about auth_time. Might be worth getting SME to check this

Reply:
@philf999 re the sensitive information, I think this is one for you

Reply:
I think the update you've added makes it a bit harder to read for me. `max_age` is the number of seconds since the last authentication that RP's are happy to accept - often zero. `auth_time` is a timestamp, represented in seconds - the current time represented that way is about `1737642273`.  When the RP makes the request, unless it's the second request in a row, they likely won't know what the `auth_time` is, as that's returned in the id_token, not the authorise response, and they don't get this until after they've authenticated

Reply:
Happy to keep to the original if adding auth_time might be confusing to users. 

Re sensitive information, my hunch is that if we keep it vague, the RP uptake might be slow as they might not realise they do/do not want to use it. One to keep an eye on and see if it's getting the adoption we're expecting